CHANGELOG.md

```diff
@@ -132,6 +132,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Enable remote segment upload backpressure by default ([#10356](https://github.com/opensearch-project/OpenSearch/pull/10356))
 - [Remote Store] Add support to reload repository metadata inplace ([#9569](https://github.com/opensearch-project/OpenSearch/pull/9569))
 - [Metrics Framework] Add Metrics framework. ([#10241](https://github.com/opensearch-project/OpenSearch/pull/10241))
+- Refactor Remote Store Metadata Lock Manager Utils ([#10379](https://github.com/opensearch-project/OpenSearch/pull/10379))
```

Comment:
We should add the actual issue this is fixing which is the separator character related